srcpkgs/routinator/files/routinator/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec routinator --config /etc/routinator/routinator.conf server --user=_routinator --group=_routinator 2>&1
```

Comment:
If this logs something, having a logging service would be nice. Look at the dockerd service in the moby package for an example.


```suggestion
[ -r conf ] && . ./conf
exec routinator --config ${CONF_FILE:-/etc/routinator/routinator.conf} server --user=_routinator --group=_routinator $OPTS 2>&1
```

Reply:
Good idea! I have included the suggestion in the new commit. Seems to work ok from what I can see.